src/common/github.js

```diff
@@ -2,22 +2,37 @@
  * This file contains helper methods that create new GitHub requests
  */
 
+import fetch from 'isomorphic-unfetch';
+
 // API URL for the issues of the repo, e.g. https://api.github.com/repos/AlexsLemonadeStand/refinebio/issues
 const GITHUB_URL = process.env.GITHUB_URL;
 // Personal access token (https://github.com/settings/tokens)
 const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 
-// Sends an issue to GitHub
+// Sends an issue to GitHub, returns true/false depending on if the fetch worked
 export async function createIssue(params) {
-  return fetch(GITHUB_URL, {
+  // fetch will give a different error if the URL is undefined, so check that first
+  if (!GITHUB_URL) {
+    return false;
+  }
+
+  fetch(GITHUB_URL, {
     method: 'POST',
     headers: {
       Accept: 'application/vnd.github.v3+json',
       Authorization: `token ${GITHUB_TOKEN}`,
       'Content-Type': 'application/json',
     },
     body: JSON.stringify(params),
-  });
+  })
+    .then(() => {
+      return true;
+    })
+    .catch(() => {
+      return false;
+    });
+
+  return false;
 }
```

Comment:
i think you will want to write this so that it is async await then check the contents and return true/fale